var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -22,8 +22,9 @@ class PyYt(PythonPackage):
 
     maintainers = ["qobilidop"]
 
-    version("develop", branch="master")
-    version("develop-4.0", branch="yt-4.0")
+    version("main", branch="main")
```

Comment:
Dependencies on main are quite different than the current deps, should probably update them, not sure what release they'll land in though. If you add the latest release, it might have the same deps. This is optional though, your change isn't making the package any worse, it was already incorrect to begin with. This could happen in a separate PR.

Reply:
I've updated to support 4.1.2 instead.